waku/v2/protocol/lightpush/waku_lightpush.go

```diff
@@ -215,31 +215,38 @@ func (wakuLP *WakuLightPush) Stop() {
 	wakuLP.h.RemoveStreamHandler(LightPushID_v20beta1)
 }
 
-// PublishToTopic is used to broadcast a WakuMessage to a pubsub topic via lightpush protocol
-func (wakuLP *WakuLightPush) PublishToTopic(ctx context.Context, message *wpb.WakuMessage, topic string, opts ...Option) ([]byte, error) {
+// Optional PublishToTopic is used to broadcast a WakuMessage to a pubsub topic via lightpush protocol
+// If pubSubTopic is not provided, then contentTopic is use to derive the relevant pubSubTopic via autosharding.
+func (wakuLP *WakuLightPush) PublishToTopic(ctx context.Context, message *wpb.WakuMessage, pubsubTopic string, opts ...Option) ([]byte, error) {
```

Comment:
Since pubsubTopic is optional, what do you think of making it an `Option` like... `withPubsubTopic`?

Reply:
I was thinking about something like this yesterday when i was working on peer selection logic, but would like to do at some common-Place as this option could be used in many places in protocols and peerManager etc.

Reply:
Looks like that is not possible, we just have to have this option specified wherever required.